### DISCUSSION TITLE
ConformalPrediction for ARIMA/AutoARIMA?

### DISCUSSION THREAD

**Discussion by linaamic** [alan-turing-institute/sktime#2869] [Q&A]
Hi everyone, I am currently trying out the ConformalIntervals feature from sktime.forecasting. However, I can't get it to work for both ARIMA and AutoARIMA. Is there an inherent difference between ARIMA forecasters and others? Do I have to call or wrap them differently or is ConformalPrediction not possible with ARIMA?
With other forecasters I have no issues.

This is the error I get:
File "...\site-packages\statsmodels\tsa\statespace\sarimax.py", line 843, in _conditional_sum_squares
    Y = endog[r:]
IndexError: too many indices for array: array is 0-dimensional, but 1 were indexed

Code:
```python
from sktime.datasets import load_airline
from sktime.forecasting.arima import AutoARIMA, ARIMA
from sktime.forecasting.conformal import ConformalIntervals

y = load_airline()
trial_forecaster = AutoARIMA()
conformal_forecaster = ConformalIntervals(trial_forecaster, method="conformal")
conformal_forecaster.fit(y, fh=[1,2,3])
```

Any help is much appreciated!

**Comment by fkiraly** [comment]
I can confirm this, python 3.8, windows, current dev, this is a bug. FYI @bethrice44.

Moving this to the issue tracker.

**Reply by fkiraly** [reply]
bug issue here: https://github.com/alan-turing-institute/sktime/issues/2889

**Reply by fkiraly** [reply]
has some workaround code, we´ll investigate the bug
